template/golang-middleware/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"handler/function"
+	//"github.com/openfaas-incubator/golang-http-template/template/golang-middleware/function"
+)
+
+func main() {
+	s := &http.Server{
+		Addr:           fmt.Sprintf(":%d", 8081),
+		ReadTimeout:    3 * time.Second,
+		WriteTimeout:   3 * time.Second,
+		MaxHeaderBytes: 1 << 20, // Max header of 1MB
+	}
+
+	http.HandleFunc("/", function.Handle)
```

Comment:
Does this support other paths?

Reply:
Nop I want to keep it simple

Reply:
(Are you sure?) Have you tested other paths?

Reply:
Seeing as this is the base template handler it should handle any path. (I think it will do that, but you should test it i.e. curl to /test/resource)

Reply:
I've tested your change and it does support other HTTP paths. This is a basic requirement of all templates now since we merged path support.